deploy/osd-managed-resources/managed-resources.ConfigMap.yaml

```diff
@@ -0,0 +1,491 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: managed-resources-map
+  namespace: openshift-monitoring
```

Comment:
Is there a way to make this configurable from the script?

Reply:
I've added options to let both of these be specified for the script.

Reply:
thanks!